src/ngraph/op/lrn.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "ngraph/op/lrn.hpp"
 #include "ngraph/op/constant.hpp"
+#include "ngraph/op/constant.hpp"
```

Comment:
This header is already included on line 18

Reply:
Redundant include was removed